docs/user-guide/grpc.md

```diff
@@ -0,0 +1,148 @@
+# gRPC example
+
+This section explains how to use Traefik as reverse proxy for gRPC application with self-signed certificates.
+
+!!! warning
+    As gRPC needs HTTP2, we need valid HTTPS certificates on both gRPC Server and Træfik.
+
+<p align="center">
+<img src="/img/grpc.svg" alt="gRPC architecture" title="gRPC architecture" />
```

Comment:
Doesn't it need to be `/docs/img/grpc.svg`? 🤔

Reply:
it's relative to web root e.g. `docs/`, see https://raw.githubusercontent.com/containous/traefik/master/docs/index.md

Reply:
https://deploy-preview-2108--traefik-docs.netlify.com/user-guide/grpc/

Reply:
:+1: